packages/component-library/src/_Themes/BrandTheme.js

```diff
@@ -0,0 +1,120 @@
+// Use with emotion's Global component
+
+/* Add all of this and BRANDING to theme for 2018 */
```

Comment:
Will something else be added to the 2018 package other than this theme?

Reply:
Ah, those were my notes from when we chatted. I need to keep digging through to make sure I've found all the style sources we're using though.